design/one-pager-release-process.md

```diff
@@ -0,0 +1,172 @@
+
+* **build/publish**: Currently, builds from the release branch will automatically run for new commits as part of the [`crossplane/crossplane`](https://jenkinsci.upbound.io/blue/organizations/jenkins/crossplane%2Fbuild/activity) pipeline.
+However, this pipeline does not correctly identify the release version, so its publishing step does not result in the correct semver container image being published.
+It is believed that [#330](https://github.com/crossplaneio/crossplane/issues/330) may be the root cause for this and we should fix this issue to get automatic publishing of release builds.
+
+* **tag**: This new pipeline would be run on the `master` branch when we have identified the release candidate commit and are ready to tag it and create the release branch.
+This pipeline would run the following build command:
+
+    ```bash
+    ./build/run make tag VERSION=${params.version}
+    ```
+
+* **promote**: This new pipeline would perform the promotion of a release build to a particular channel.
+We have currently been using `master` and `alpha` channels, and we can consider in the future changing to other channels such as `stable` and `edge`, but that is out of scope of this document.
+This pipeline would run the following build command:
+
+    ```bash
+    ./build/run make promote BRANCH_NAME=${BRANCH_NAME} VERSION=${params.version} CHANNEL=${params.channel}
+    ```
+
+## Stack Releases
+
+While this document has so far described the release process improvements for the [main Crossplane repository](https://github.com/crossplaneio/crossplane),
+there are also some infrastructure stacks that the Crossplane community currently owns and is responsible for releasing, such as [`stack-gcp`](https://github.com/crossplaneio/stack-gcp).
+Since these stack repositories currently also take advantage of the build submodule, they will also follow a very similar process as Crossplane and receive a similar set of improvements.
+
+Recall that a Stack is simply just an [OCI image](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#stack-package-format).
+Therefore, the build and publishing process can follow essentially the same process that Crossplane uses.
+Other functionality in the build submodule, such as creating a Helm chart, is not necessary for Stacks, but relevant functionality should definitely be reused.
+For example, building and publishing the stack package, tagging and creating a release branch, promoting the release, etc.
+
+The GCP, AWS, and Azure stacks will all receive new `tag` and `promote` pipelines similar to Crossplane, as well as receive a fix for [#330](https://github.com/crossplaneio/crossplane/issues/330).
+
+Note that each Stack can be released independently and on their own schedule, but for now they will follow the same monthly release schedule as Crossplane so the community can receive frequent updates.
```

Comment:
Will each stack receive patch releases with Crossplane? Can we do patch releases of individual stacks out of cadence with Crossplane and other stacks?

Reply:
Yes, we can do patch releases for stacks completely on their own schedule as well.  They can also even NOT have a patch release while other stacks or crossplane does get one.  I'll add a note for that.